TWLight/resources/templates/resources/partner_tile.html

```diff
@@ -76,7 +76,7 @@
       </div>
       <hr />
     {% endif %}
-    <div class="panel-partner-info">
+    <!-- <div class="panel-partner-info">
```

Comment:
We want to remove this, so instead of commenting out you can simply delete the content!